fluidsf/generate_structure_functions_1d.py

```diff
@@ -0,0 +1,167 @@
+import numpy as np
+
+from .bin_data import bin_data
+from .calculate_separation_distances import calculate_separation_distances
+from .calculate_structure_function_1d import calculate_structure_function_1d
+from .shift_array1d import shift_array1d
+
+
+def generate_structure_functions_1d(  # noqa: C901, D417
+    u,
+    v,
```

Comment:
Technically `v` is only necessary for `LTT` structure functions, so it should probably be optional.

Reply:
Added to 445539a.